tools/ci_build/github/linux/docker/Dockerfile.ubuntu_cuda11_4_tensorrt7_2

```diff
@@ -0,0 +1,43 @@
+# --------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------
+# Dockerfile to run ONNXRuntime with TensorRT integration
+
+ARG OS_VERSION=18.04
+FROM nvidia/cuda:11.4.0-cudnn8-devel-ubuntu${OS_VERSION}
+
+ARG TRT=7.2.3.4
+ENV CUDA_VERSION=11.1
```

Comment:
this isn't really the CUDA VERSION being used here. 

 can we just specify the TRT version string like 

and https://github.com/microsoft/onnxruntime/blob/master/tools/ci_build/github/linux/docker/Dockerfile.manylinux2014_cuda11_4_tensorrt8_0#L180